networkx/algorithms/centrality/tests/test_decay_centrality.py

```diff
@@ -0,0 +1,327 @@
+"""
+Unit tests for decay centrality
+
+"""
+
+import pytest
+
+import networkx as nx
+
+
+def test_decay_centrality_K():
```

Comment:
rename the tests to describe what they are testing: e.g. `test_decay_centrality_kite_graph`
`test_decay_centrality_tree`
`test_decay_centrality_mode_inputs`
etc